.changeset/tiny-starfishes-mix.md

```diff
@@ -0,0 +1,7 @@
+---
+'@kadena/graph-client': patch
+'@kadena/graph': patch
+---
+
+Added signer object and signer fields on transactions (graph); Changed the
+transaction overview page to display signer related information
```

Comment:
Nitpick: you can add two lines with - in front, then on the changelog it'll show as two parts. Maybe even better to have the two separated.

1. ```
   ---
   '@kadena/graph-client': patch
   '@kadena/graph': patch
   ---
   
   Added signer object and signer fields on transactions (graph)
   ```


2. ```
   ---
   '@kadena/graph-client': patch
   ---
   
   Changed the transaction overview page to display signer related information
   ```